client/src/bundles/bundle.js

```diff
@@ -81,6 +81,7 @@ import 'expose-loader?exposes=formatWrittenNumber!lib/formatWrittenNumber';
 import 'expose-loader?exposes=withDragDropContext!lib/withDragDropContext';
 import 'expose-loader?exposes=withRouter!lib/withRouter';
 import 'expose-loader?exposes=ssUrlLib!lib/urls';
+import 'expose-loader?exposes=SearchableDropdownField!components/SearchableDropdownField/SearchableDropdownField';
```

Comment:
Exposing things through here needs to be coupled with a PR to the webpack config, or people can't use it.

Reply:
https://github.com/silverstripe/webpack-config/pull/64